rust/processor/parser.yaml

```diff
@@ -1,7 +1,7 @@
 # This is a template yaml for the indexer-grpc parser.
 health_check_port: 8084
 server_config:
-    processor_name: default_processor
-    postgres_connection_string: postgresql://postgres:@localhost:5432/postgres_v2
-    indexer_grpc_data_service_address: 127.0.0.1:50051:50051
-    auth_token: AUTH_TOKEN
\ No newline at end of file
+  processor_name: default_processor
+  postgres_connection_string: postgresql://postgres:@localhost:5432/postgres_v2
+  indexer_grpc_data_service_address: 127.0.0.1:50051:50051
```

Comment:
this is a typo lol

Reply:
this is just reformatting

Reply:
No I mean the 50051:50051. Remove one of them (line 6)